doc/manual/src/contributing/documentation.md

```diff
@@ -196,15 +196,17 @@ You can also build and view it yourself:
 [Doxygen API documentation]: https://hydra.nixos.org/job/nix/master/internal-api-docs/latest/download-by-type/doc/internal-api-docs
 
 ```console
-# nix build .#hydraJobs.internal-api-docs
-# xdg-open ./result/share/doc/nix/internal-api/html/index.html
+$ nix build .#hydraJobs.internal-api-docs
+$ xdg-open ./result/share/doc/nix/internal-api/html/index.html
```

Comment:
By convention, `#` is for root shells, while `$` is for regular user shells. (I wish I didn't know this silly Unix lore :D)